bin/ci-test

```diff
@@ -37,8 +37,6 @@ function build() {
   export CORE=dependabot/dependabot-core
   export UPDATER=dependabot/dependabot-updater
 
-  # shellcheck disable=SC2034  # shellcheck isn't smart enough to realize `docker build` uses `TARGETARCH`
-  local TARGETARCH;
```

Comment:
As suggested by @deivid-rodriguez in https://github.com/dependabot/dependabot-core/pull/6478#discussion_r1081460417

Reply:
Just want to note the difference with specifying `local`. Without `local`, `TARGETARCH` becomes available outside of the function. IMO, not a big deal.